libs/sdk-ui-filters/src/AttributeFilter/AttributeFilter.tsx

```diff
@@ -64,8 +69,19 @@ export interface IAttributeFilterProps {
 
     /**
      * Specify a parent attribute filter that will be used to reduce options for available components options.
+     *
+     * Parent filters MUST be defined by uris.
```

Comment:
This should be more concrete. the parent filter's attribute elements must contain URI's right? or even the display form must be URI?